contrib/build_rpm.sh

```diff
@@ -70,3 +72,21 @@ if [ -d ~/rpmbuild/BUILD ]; then
 fi
 
 rpmbuild --rebuild ${extra_arg:-} podman-*.src.rpm
+
+# builds build/podman.repo which should be ready to install locally
+mkdir -p build/buildset
+rm -rf build/buildset/*
+pushd build/buildset
+   cp -l ~/rpmbuild/RPMS/*/*.rpm .
+   createrepo .
+popd
+cat <<EOF >build/podman.repo
```

Comment:
This is probably not absolutely critical, but the correct way to create files is to make a tempfile and rename it on success:
```
cat <<EOF >build/podman.repo.tmp.$$ && mv build/podman.repo.tmp.$$ build/podman.repo
```
With this form, if `build/podman.repo` exists it is guaranteed to be correct; without a tmp-rename, it is possible for an error (typically out-of-space but also possibly i/o error) to leave a corrupt file in place. This is a good habit to get into.